torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1935,6 +1935,32 @@ def sample_inputs_nn_functional_prelu(op_info, device, dtype, requires_grad, **k
     yield SampleInput(make_arg((S, S)), kwargs=dict(weight=weight_tensor,))
     yield SampleInput(make_arg((S, S)), kwargs=dict(weight=make_arg((S,)),))
 
+def sample_kwargs_prelu_scalar_weight(device, dtype, input):
```

Comment:
Add a comment for what's up with NumPy -- can't the NumPy weight be float32 but the torch weight still in bfloat16?